docs(AwsRedisInstance): add readReplica details (#762)
kyma-project/cloud-manager: docs/user/resources/04-70-aws-redis-instance.md

| **cacheNodeType**                                 | string | Required. A node is the smallest building block of an Amazon ElastiCache deployment. It is a fixed-size chunk of secure, network-attached RAM. For supported node tyes, check [Amazon's Supported node types page](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheNodes.SupportedTypes.html) |
| **engineVersion**                                 | string | Optional. The version number of the cache engine to be used for the clusters in this replication group. To see all supported versions, check [Amazon's Supported ElastiCache (Redis OSS) versions page](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/supported-engine-versions.html). Defaults to `"7.0"`. |
| **authEnabled**                                   | bool   | Optional. Enables using an AuthToken (password) when issuing Redis OSS commands. Defaults to `false`.                                                                                                       |

> | **readReplicas**                                  | number | Optional. Number of read replicas. If greater than zero, automatic failover will be enabled to ensure instance high availability. Supported values are `0`, and `1`. Defaults to `0`.                       |

Reviewer: ```suggestion
| **readReplicas**                                  | number | Optional. Number of read replicas. If greater than zero, automatic failover is enabled to ensure the high availability of the instance. Supported values are `0` and `1`. Defaults to `0`.                       |
```
